src/midi/midi_mt32.h

```diff
@@ -71,18 +66,18 @@ class MidiHandler_mt32 final : public MidiHandler {
 
 	// Managed objects
 	channel_t channel{nullptr, MIXER_DelChannel};
-	limited_buffer_t buffer{};
+	std::vector<int16_t> play_buffer{};
```

Comment:
I think this buffer should be initialized to a specific size in `MidiHandler_mt32` c-tor initialization list?

Reply:
The `play_buffer` never sized; it's only ever assigned when dequeue'ing from the ring 
(rqueue `std::move` the content into it, but otherwise this is an assign in our less sophisticated queue too).

The stuff that's assigned into it comes from the SoftLimiter's output (`auto out = limiter... `), which is enqueued into the ring.

The ring only transacts in enqueued objects, so at least one entire rendering pass and enqueue needs to complete, (the ring never hands back and empty or default `T` if it's empty); instead it just waits and blocks.

Reply:
Infact, this is a good question for you:

Is there any way we can instantiate `play_buffer` as a container-for-movement-only? ie: it will only ever be `moved` in.  Something like `std::vector<int16_t> &&play_buffer = {};`?

Reply:
Fortunately it's not possible.

Reply:
> Fortunately it's not possible.

:sweat_smile: